2019-11-03 21:36:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-196]
2019-11-03 21:36:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-196]
2019-11-03 21:36:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-196]
2019-11-03 23:45:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "USERINFO" not found; SQL statement:
UPDATE userinfo.myusers SET name='Zoya' where id=2; [90079-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.getSchema(Parser.java:688)
	at org.h2.command.Parser.getSchema(Parser.java:694)
	at org.h2.command.Parser.readTableOrView(Parser.java:5535)
	at org.h2.command.Parser.readTableOrView(Parser.java:5529)
	at org.h2.command.Parser.readSimpleTableFilter(Parser.java:796)
	at org.h2.command.Parser.parseUpdate(Parser.java:739)
	at org.h2.command.Parser.parsePrepared(Parser.java:471)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:132)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:121)
	at org.DbOperations.update(DbOperations.scala:16)
	at org.UserOperations.updateUser(UserOperations.scala:19)
	at org.UserOperationsSpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(UserOperationsSpec.scala:25)
	at org.UserOperationsSpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(UserOperationsSpec.scala:24)
	at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
	at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
	at org.scalatest.Transformer.apply(Transformer.scala:22)
	at org.scalatest.Transformer.apply(Transformer.scala:20)
	at org.scalatest.WordSpecLike$$anon$1.apply(WordSpecLike.scala:1078)
	at org.scalatest.TestSuite$class.withFixture(TestSuite.scala:196)
	at org.scalatest.WordSpec.withFixture(WordSpec.scala:1881)
	at org.scalatest.WordSpecLike$class.invokeWithFixture$1(WordSpecLike.scala:1075)
	at org.scalatest.WordSpecLike$$anonfun$runTest$1.apply(WordSpecLike.scala:1088)
	at org.scalatest.WordSpecLike$$anonfun$runTest$1.apply(WordSpecLike.scala:1088)
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:289)
	at org.scalatest.WordSpecLike$class.runTest(WordSpecLike.scala:1088)
	at org.scalatest.WordSpec.runTest(WordSpec.scala:1881)
	at org.scalatest.WordSpecLike$$anonfun$runTests$1.apply(WordSpecLike.scala:1147)
	at org.scalatest.WordSpecLike$$anonfun$runTests$1.apply(WordSpecLike.scala:1147)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:396)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:384)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:373)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:410)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:384)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:379)
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:461)
	at org.scalatest.WordSpecLike$class.runTests(WordSpecLike.scala:1147)
	at org.scalatest.WordSpec.runTests(WordSpec.scala:1881)
	at org.scalatest.Suite$class.run(Suite.scala:1147)
	at org.scalatest.WordSpec.org$scalatest$WordSpecLike$$super$run(WordSpec.scala:1881)
	at org.scalatest.WordSpecLike$$anonfun$run$1.apply(WordSpecLike.scala:1192)
	at org.scalatest.WordSpecLike$$anonfun$run$1.apply(WordSpecLike.scala:1192)
	at org.scalatest.SuperEngine.runImpl(Engine.scala:521)
	at org.scalatest.WordSpecLike$class.run(WordSpecLike.scala:1192)
	at org.UserOperationsSpec.org$scalatest$BeforeAndAfterAll$$super$run(UserOperationsSpec.scala:8)
	at org.scalatest.BeforeAndAfterAll$class.liftedTree1$1(BeforeAndAfterAll.scala:213)
	at org.scalatest.BeforeAndAfterAll$class.run(BeforeAndAfterAll.scala:210)
	at org.UserOperationsSpec.run(UserOperationsSpec.scala:8)
	at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:314)
	at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:480)
	at sbt.TestRunner.runTest$1(TestFramework.scala:139)
	at sbt.TestRunner.run(TestFramework.scala:154)
	at sbt.TestFramework$$anon$3$$anonfun$$lessinit$greater$1.$anonfun$apply$1(TestFramework.scala:317)
	at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:277)
	at sbt.TestFramework$$anon$3$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:317)
	at sbt.TestFramework$$anon$3$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:317)
	at sbt.TestFunction.apply(TestFramework.scala:329)
	at sbt.Tests$.processRunnable$1(Tests.scala:350)
	at sbt.Tests$.$anonfun$makeSerial$1(Tests.scala:356)
	at sbt.std.Transform$$anon$3.$anonfun$apply$2(Transform.scala:46)
	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:280)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
	at sbt.Execute.work(Execute.scala:289)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:280)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2019-11-03 23:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-03 23:48:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-03 23:51:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-03 23:56:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-03 23:57:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-04 21:27:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-04 21:32:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-07 16:13:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-07 16:18:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-07 16:22:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-07 16:23:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-07 16:24:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-07 16:25:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
2019-11-07 16:29:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-196]
